lib/xcodeproj/workspace.rb

```diff
@@ -67,9 +89,36 @@ def self.from_s(xml, workspace_path = '')
     # @return [void]
     #
     def <<(projpath)
```

Comment:
Now that the parameter to this method can be either a `String` or a `FileReference`, please update the rdoc accordingly